sonar-ws/src/main/java/org/sonarqube/ws/client/setting/SettingsService.java

```diff
@@ -61,6 +63,9 @@ public void set(SetRequest request) {
     call(new PostRequest(path(ACTION_SET))
       .setParam(PARAM_KEY, request.getKey())
       .setParam(PARAM_VALUE, request.getValue())
+      // TODO doesn't work for multi param
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)